docs/sources/zoom/README.md

```diff
@@ -81,3 +81,21 @@ sufficient, but as of May 2024 are no longer available for newly created Zoom ap
 Once the scopes are added, click on `Done` and then `Continue`.
 
 6. Activate the app
```

Comment:
Maybe you can include an screenshot like this on this step too:

![image](https://github.com/user-attachments/assets/7c9fabba-b96e-41fe-b9cf-37d6eec33c18)